sci-libs/nipy-data/nipy-data-0.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils multilib flag-o-matic
+
+DESCRIPTION="Neuroimaging tools for Python"
+HOMEPAGE="http://nipy.org/"
+SRC_URI="http://nipy.org/data-packages/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86 amd64-linux x86-linux"
```

Comment:
It's a colection of rather static example data files. afaict they have not been modified for a long tim, nor will they be modified in the future. There is really nothing about them that can be unstable.